tests/tests_lvm_pool_shared.yml

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
I guess this entire test case should be skipped with older versions of blivet that don't support shared LVM. The support is not yet released on CentOS Stream/RHEL yet, but it should be available in Blivet 3.6.0-11 (EL9) and 3.6.0-8 (EL8) and in Fedora in Blivet 3.8.2-1 or newer.